README.md

```diff
@@ -256,3 +259,23 @@ These modelines are more accurate giving exact hz. However, some games may have
 Some arcade resolutions can be very different from consumer CRTs. There is resolution detection to ensure MAME games will be displayed in the closest available resolution but drawn at their native resolution within this resolution. Meaning that the MAME game will look just like the original hardware.
 
 MAME ROMs that run in a vertical aspect like DoDonPachi need to be rotated within MAME before resolution switching and aspect correction will work. Do this before enabling CRT SwitchRes so that RetroArch will run in your desktop resolution. Once you have rotated any games that may need it turn CRT SwitchRes on.
+
+## Socials
+
+The links below belong to our official channels. Links other than this may have been created by fans, independent members or followers. We seriously recommend using our original resources.
+
+- Website: https://www.retroarch.com/
+- Blog: http://libretro.com/
+- Facebook: https://www.facebook.com/libretro
+- Twitter: https://twitter.com/libretro
+- Reddit: https://www.reddit.com/r/RetroArch/
+- YouTube: https://www.youtube.com/Libretro
+- Google Post: https://posts.google.com/share/55Nhs2jG
+- Steam: https://store.steampowered.com/app/1118310/RetroArch/
+- YouTube Topic: https://www.youtube.com/channel/UC5q007PYyQPgin0HHbzF0zQ
+- Patreon: https://www.patreon.com/libretro
+- BOUNTYSOURCE: https://www.bountysource.com/teams/libretro/issues
+- Discord: https://discord.gg/27Xxm2h
+- Teespring: https://teespring.com/stores/retroarch
+- Documentation: http://docs.libretro.com/
+- Forums: https://forums.libretro.com/
```

Comment:
Could probably use Markdown links here. For example:
```
- [YouTube](https://www.youtube.com/Libretro)
```

Reply:
I wasn't sure if it look good so i didn't make it as Markdown in first place.